src/Microsoft.Identity.Client/ApiConfig/AcquireTokenWithIntegratedWindowsAuthParameterBuilder.cs

```diff
@@ -67,7 +71,13 @@ public AcquireTokenWithIntegratedWindowsAuthParameterBuilder WithUsername(string
         /// <inheritdoc />
         internal override Task<AuthenticationResult> ExecuteAsync(IPublicClientApplicationExecutor executor, CancellationToken cancellationToken)
         {
-            return executor.ExecuteAsync((IAcquireTokenWithIntegratedWindowsAuthParameters)Parameters, cancellationToken);
+            return executor.ExecuteAsync(CommonParameters, Parameters, cancellationToken);
+        }
+
+        /// <inheritdoc />
+        internal override ApiEvent.ApiIds CalculateApiEventId()
+        {
+            return ApiEvent.ApiIds.AcquireTokenWithScopeUser;
```

Comment:
>ScopeUser [](start = 51, length = 9)

what's ScopeUser?

Reply:
AcquireToken, specifying scopes, and specifying an account/user.

---
In reply to: [248925578](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/798#discussion_r248925578) [](ancestors = 248925578)

Reply:
thanks.

---
In reply to: [249130049](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/798#discussion_r249130049) [](ancestors = 249130049,248925578)